megamek/src/megamek/common/MechSummaryCache.java

```diff
@@ -236,6 +256,15 @@ public void loadMechData(boolean ignoreUnofficial) {
             }
         }
 
+        checkForChanges(ignoreUnofficial, vMechs, sKnownFiles, lLastCheck);
+        updateData(vMechs);
+        addLookupNames();
+        logReport();
+
+        done();
+    }
+
+    private void checkForChanges(boolean ignoreUnofficial, Vector<MechSummary> vMechs, Set<String> sKnownFiles, long lLastCheck) {
```

Comment:
Too long, let's split this into two lines

Reply:
Also, by the name I would assume it returns a boolean. Is there a reason this doesn't?

Reply:
I don't know why you would think that. There's no value to return.

Reply:
I would assume checkForChanges checks for any changes, but doesn't do any updates to it, just based on the name. Maybe add a comment saying it also updates, or rename?